app/src/main/java/org/p2p/wallet/bridge/api/mapper/BridgeMapper.kt

```diff
@@ -74,7 +75,7 @@ class BridgeMapper {
 
     fun fromNetwork(response: BridgeSendTransactionResponse): BridgeSendTransaction {
         return BridgeSendTransaction(
-            transaction = response.transaction,
+            transaction = response.transaction.toBase64String(),
```

Comment:
If you receive Base64 from response, you can declare transaction as Base64String in Response model